lms/static/js/spec/edxnotes/views/notes_visibility_factory_spec.js

```diff
@@ -70,6 +70,16 @@ define([
             expect(this.toggleMessage).toContainText('Notes visible');
             expect(Annotator._instances).toHaveLength(2);
 
+            // TODO: why is the same search request made twice?
```

Comment:
Can you create a bug for this in the epic https://openedx.atlassian.net/browse/TNL-2461?